lib/samples/identify_raster_cell/README.md

```diff
@@ -0,0 +1,35 @@
+# Identify raster cell
+
+Get the cell value of a local raster at the tapped location and display the result in a callout.
```

Comment:
```suggestion
Get the cell value of a local raster at the tapped location and display the result in the UI.
```
Callout is a specific API that we will eventually implement but don't support yet.